src/components/Navbar/index.tsx

```diff
@@ -15,10 +15,11 @@ export function renderNavLink(
 ): React.ReactFragment {
   const { title, slug } = link;
   const pagePaths = pathname.split("/");
+  const isAksara = pagePaths[1] === "aksara";
```

Comment:
Instead of doing this, I suggest doing this instead:

In the `Navbar/links` file, can add a new element to the links objects
something like this `{title: "Aksara", slug: "aksara", alt: true}`

If alt is true, render Navbar with alternate colors/themes

This is to make the component as generic and as uncoupled from specificities like slug names as much as possible

Reply:
done